portal/config/eproms/Organization.json

```diff
@@ -1671,6 +1671,67 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/valueset/languages",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "code": "en_GB",
+                "display": "UK English",
```

Comment:
Can use you an existing `display` value?

Looks like we have [`English (United Kingdom)` ](https://github.com/uwcirg/true_nth_usa_portal/blob/91aff5c423e13a91c9cec3de5e7c27d4095530f5/portal/config/eproms/Coding.json#L6064-L6067)and [`English`](https://github.com/uwcirg/true_nth_usa_portal/blob/b63f4060862dc5f1f2068ca54467cc993d0b5917/portal/config/eproms/Organization.json#L1496-L1498) for that locale (`en_GB`)

Reply:
done, thanks